app/(legacy)/projects/page.tsx

```
@@ -0,0 +1,111 @@
+              metadata={{
+                data: {
+                  bio: project.fractions.data[0].metadata.description,
+                  impactCategory:
+                    project.fractions.data[0].metadata.impact_scope,
+                },
+              }}
+              isLoading={false}
+              buttonText="Manage"
+              buttonLink={`/manage/${project.hypercert_id}`}
+            />
+          ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)